test/index.test.js

```diff
@@ -49,46 +49,26 @@ describe('Keen IO', function() {
     });
 
     describe('#initialize', function() {
-      it('should create window.Keen', function() {
-        analytics.assert(!window.Keen);
+      it('should create window.KeenSegment', function() {
+        analytics.assert(!window.KeenSegment);
         analytics.initialize();
         analytics.page();
-        analytics.assert(window.Keen);
+        analytics.ready(function() {
+          analytics.assert(window.KeenSegment);
+        });
       });
 
-      it('should configure Keen', function() {
+      it('should configure KeenSegment', function() {
         analytics.initialize();
         analytics.page();
-        analytics.deepEqual(keen.client._config, {
-          projectId: options.projectId,
-          writeKey: options.writeKey,
-          readKey: ''
+        analytics.ready(function() {
+          analytics.equal(keen.client.projectId(), options.projectId);
+          analytics.equal(keen.client.writeKey(), options.writeKey);
         });
       });
     });
   });
 
-  describe('loading', function() {
```

Comment:
Can we also add a test somewhere that checks that whole behavior regarding `previousKeen`?